### PR TITLE
Figure edits

### PR DESCRIPTION
**Pull request by scottdtaylor95**

### CODE REVIEW

**Comment by scottdtaylor95** [comment]
@aarmey All the changes have been made except I wanted to ask a question before doing a caption for fig S2. I'm a little unclear on the data - what exactly does it represent? My idea was something along the lines of - antibodies were taken from HIV patients in the 4 categories and run through some tests. I assume the antibodies were separated somehow to isolate only those for HIV, but I'm not sure how, nor if they isolated further for specific antigens. The function data is from certain functional assays run with the antibodies, I assume in an in vitro cellular environment. The luminex data is what I'm least sure of, both the regular and the igg. Did they isolate antibodies for each antigen (if so, how?) and then use beads covered with the various detections? How does that work for the IgG detections?

**Comment by aarmey** [comment]
I can write the caption for S2 in that case. I'll tag you so you can see it. This paper also describes the measurements: https://pubmed.ncbi.nlm.nih.gov/23023091/

**Comment by scottdtaylor95** [comment]
Thanks, this should help. Also I left a few places that might need references for python packages or the Alter paper/this other paper for the data.

File: syserol/figures/figureS1.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def makeFigure():
         ax[i].set_ylabel("Predicted Values")
         m, b = np.polyfit(x, y, 1)  # line of best fit
         ax[i].plot(x, m * x + b, 'k--', color="red")
-        ax[i].text(1, 1, f"Accuracy Score: {round(accuracy, 3)}", {"color": "red", "fontsize": 10}, horizontalalignment="right",
```
Comment (aarmey) [inline syserol/figures/figureS1.py:25]: I adjusted this. Please check its placement before merging.

**Review by aarmey — APPROVED** [review]
Great! Just check the one item I commented on before merging.

File: syserol/figures/figureS1.py (hunk 1 above)
Comment (scottdtaylor95) [reply syserol/figures/figureS1.py:25]: Looks good to me, I'll merge now

**Pull request merged**